src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.tsx

```diff
@@ -154,25 +154,18 @@ export function createContextCommandBarButtons(container: Explorer): CommandButt
 }
 
 export function createControlCommandBarButtons(container: Explorer): CommandButtonComponentProps[] {
-  const buttons: CommandButtonComponentProps[] = [];
-  if (configContext.platform === Platform.Hosted) {
-    return buttons;
-  }
-
-  if (!container.isPreferredApiCassandra()) {
```

Comment:
Whatabout this? Does it matter?

Reply:
Nope. We can show it there too. It should be present for Cassandra users. I think this is just legacy